gradle/dependency-versions.gradle

```diff
@@ -17,4 +17,5 @@ ext {
     commonsVersion = "1.5"
     bouncycastleVersion = "1.52"
     javaxVersion = "3.0.1"
+    helixVersion = "0.6.7"
```

Comment:
this is the latest version as of this writing from the 0.6.x branch, which is the stable branch. This version itself has not yet been officially announced, but this has the changes we need.